charts/policy-controller/README.md

```diff
@@ -27,8 +27,8 @@ The Helm chart for Policy  Controller
 | cosign.webhookName | string | `"policy.sigstore.dev"` |  |
 | imagePullSecrets | list | `[]` |  |
 | installCRDs | bool | `true` |  |
+| policywebhook.configData | object | `{}` |  |
```

Comment:
let keep the old description ` Set the data of the `policy-config-controller` configmap`.